docs/tutorials/work_with_lsdb.ipynb

```diff
@@ -7,7 +7,9 @@
    "source": [
```

Comment:
<p>nit: I think we can remove the "the" in front of LSDB both in the title and in the first sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>